docs/mine-hnt/validators/troubleshooting.mdx

```diff
@@ -58,8 +58,9 @@ Helium hosts snapshots for download for the convenience of syncing validators. Y
 * First determine the height of the snapshot you would like to download:
   * The list of latest snapshots which have been agreed upon by the Consensus Group is available from the API at [https://api.helium.io/v1/snapshots](https://api.helium.io/v1/snapshots)
   * Additionally, the latest manually blessed snapshot used by hotspots to sync from is available at [https://snapshots.helium.wtf/mainnet/latest-snap.json](https://snapshots.helium.wtf/mainnet/latest-snap.json). Note: this link may include snapshots generated by Helium or the Helium ETL that were not agreed upon by the Consensus Group and may not have been tested to work on validators.
-* Choose the block height you want to start from and download the snapshot from Helium:  
-  * `curl -O https://snapshots.helium.wtf/mainnet/snap-<height>`
+* Choose the block height you want to start from and download the compressed version of this snapshot from Helium:  
+  * `curl -O https://snapshots.helium.wtf/mainnet/snap-<height>.gz`
+  * `gzip -d snap-<height>.gz`
```

Comment:
No need to decompress the file first. The validator will do that.

Reply:
I've updated this branch...